hubblestack/extmods/modules/hstatus.py

```diff
@@ -11,12 +16,50 @@ def get():
 
         (probably only useful from other excution modules)
     '''
-    return hubblestack.status.HubbleStatus.as_json()
+    return hubblestack.status.HubbleStatus.stats()
+
+def msg_counts(pat=r'hubblestack.hec.obj.input:(?P<stype>[^:]+)', reset=True):
+    ''' returns counter data formatted for the splunk_generic_return returner
+
+        params:
+            pat - the key matching algorithm is a simple regular expression
+    '''
+
+    km = re.compile(pat)
+    r = list()
+    s = get()
```

Comment:
Can we rename `get()` to `get_stats()` or something? I like to be explicit and self-documenting where possible.